layered-architectures-module-2/Carting.BLL/Models/CartItem.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace Carting.BLL.Models;
+
+public class CartItem
+{
+    public int? ExternalId { get; set; }
```

Comment:
Id Name Price Quantity are required. You need to add a validation and remove nullability here

Reply:
I see it more as a buisiness rule, so that is why model can contain such values however CartItemValidator will prevent to perform create action. Because, for instance price can be 0, however this is a default value for int as well, so we don't know whether it was specified on purpose or no